examples/AzureSDKDemoSwiftUI/AzureSDKDemoSwiftUI/Utils/Common.swift

```diff
@@ -0,0 +1,183 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import AzureCore
+import AzureIdentity
+import AzureStorageBlob
+import Foundation
+import MSAL
+
+struct AppConstants {
+    static let storageAccountUrl = URL(string: "https://iosdemostorage1.blob.core.windows.net/")!
+    static let tenant = "7e6c9611-413e-47e4-a054-a389854dd732"
+    static let clientId = "6f2c62dd-d6b2-444a-8dff-c64380e7ac76"
+    static let redirectUri = "msauth.com.azure.examples.AzureSDKDemoSwift://auth"
+    static let authority = "https://login.microsoftonline.com/7e6c9611-413e-47e4-a054-a389854dd732"
+    static let uploadContainer: String! = "uploads"
+    static let videoContainer: String = "videos"
+    
+    // swiftlint:disable line_length
+    static let sasConnectionString =
```

Comment:
Add a comment here that this string is a read-only SAS token and may need to be regenerated.